### DISCUSSION TITLE
Nested Dropdown Direction

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#3166] [Q&A] [answered]
Hey,

so yet another question of mine.

As I have seen nested dropdowns have been added to Blazorise.
After using them for a bit, what I have noticed that the orientation of the nested ones can not be set, is that true?

Because unfortunately they are certainly limited, which leeds to them being drawn behind the boudaries of the browser window. (overflow isnt set correctly but this would not change a lot)

![image](https://user-images.githubusercontent.com/10349466/143434462-cca79d85-d0cc-47cf-892b-78a88b16a449.png)

Kind Regards
Sebastian

**Comment by stsrki** [comment, marked as answer]
That's true. In the current version, we only support nested menus that go to the right side. We plan to add support for other directions in the next version.

**Reply by bastigerner** [reply]
Ok cool good to know thanks!
